pallets/messages/src/lib.rs

```diff
@@ -369,48 +371,4 @@ impl<T: Config> Pallet<T> {
 			.map(|msg| msg.map_to_response(block_number_value, schema_payload_location))
 			.collect()
 	}
-
-	/// Moves messages from temporary storage `BlockMessages` into final storage `Messages`
-	/// and calculates execution weight
-	///
-	/// Called inside of `on_initialize`
-	///
-	/// # Events
-	/// * [`Event::MessagesStored`]
-	///
-	/// Returns execution weights
-	///
-	fn move_messages_into_final_storage(block_number: T::BlockNumber) -> Weight {
```

Comment:
If BlockMessage was in use we would need to drain it first before removing this. But since we don't have any messages in there directing removing this should be fine